[
  {
    "id":304538917,
    "node_id":"MDEwOlJlcG9zaXRvcnkzMDQ1Mzg5MTc=",
    "name":"CVE-2020-16898",
    "full_name":"Q1984\/CVE-2020-16898",
    "private":false,
    "html_url":"https:\/\/github.com\/Q1984\/CVE-2020-16898",
    "description":"PoC Bad Neighbor BSOD exploit",
    "fork":false,
    "url":"https:\/\/api.github.com\/repos\/Q1984\/CVE-2020-16898",
    "forks_url":"https:\/\/api.github.com\/repos\/Q1984\/CVE-2020-16898\/forks",
    "keys_url":"https:\/\/api.github.com\/repos\/Q1984\/CVE-2020-16898\/keys{\/key_id}",
    "collaborators_url":"https:\/\/api.github.com\/repos\/Q1984\/CVE-2020-16898\/collaborators{\/collaborator}",
    "teams_url":"https:\/\/api.github.com\/repos\/Q1984\/CVE-2020-16898\/teams",
    "hooks_url":"https:\/\/api.github.com\/repos\/Q1984\/CVE-2020-16898\/hooks",
    "issue_events_url":"https:\/\/api.github.com\/repos\/Q1984\/CVE-2020-16898\/issues\/events{\/number}",
    "events_url":"https:\/\/api.github.com\/repos\/Q1984\/CVE-2020-16898\/events",
    "assignees_url":"https:\/\/api.github.com\/repos\/Q1984\/CVE-2020-16898\/assignees{\/user}",
    "branches_url":"https:\/\/api.github.com\/repos\/Q1984\/CVE-2020-16898\/branches{\/branch}",
    "tags_url":"https:\/\/api.github.com\/repos\/Q1984\/CVE-2020-16898\/tags",
    "blobs_url":"https:\/\/api.github.com\/repos\/Q1984\/CVE-2020-16898\/git\/blobs{\/sha}",
    "git_tags_url":"https:\/\/api.github.com\/repos\/Q1984\/CVE-2020-16898\/git\/tags{\/sha}",
    "git_refs_url":"https:\/\/api.github.com\/repos\/Q1984\/CVE-2020-16898\/git\/refs{\/sha}",
    "trees_url":"https:\/\/api.github.com\/repos\/Q1984\/CVE-2020-16898\/git\/trees{\/sha}",
    "statuses_url":"https:\/\/api.github.com\/repos\/Q1984\/CVE-2020-16898\/statuses\/{sha}",
    "languages_url":"https:\/\/api.github.com\/repos\/Q1984\/CVE-2020-16898\/languages",
    "stargazers_url":"https:\/\/api.github.com\/repos\/Q1984\/CVE-2020-16898\/stargazers",
    "contributors_url":"https:\/\/api.github.com\/repos\/Q1984\/CVE-2020-16898\/contributors",
    "subscribers_url":"https:\/\/api.github.com\/repos\/Q1984\/CVE-2020-16898\/subscribers",
    "subscription_url":"https:\/\/api.github.com\/repos\/Q1984\/CVE-2020-16898\/subscription",
    "commits_url":"https:\/\/api.github.com\/repos\/Q1984\/CVE-2020-16898\/commits{\/sha}",
    "git_commits_url":"https:\/\/api.github.com\/repos\/Q1984\/CVE-2020-16898\/git\/commits{\/sha}",
    "comments_url":"https:\/\/api.github.com\/repos\/Q1984\/CVE-2020-16898\/comments{\/number}",
    "issue_comment_url":"https:\/\/api.github.com\/repos\/Q1984\/CVE-2020-16898\/issues\/comments{\/number}",
    "contents_url":"https:\/\/api.github.com\/repos\/Q1984\/CVE-2020-16898\/contents\/{+path}",
    "compare_url":"https:\/\/api.github.com\/repos\/Q1984\/CVE-2020-16898\/compare\/{base}...{head}",
    "merges_url":"https:\/\/api.github.com\/repos\/Q1984\/CVE-2020-16898\/merges",
    "archive_url":"https:\/\/api.github.com\/repos\/Q1984\/CVE-2020-16898\/{archive_format}{\/ref}",
    "downloads_url":"https:\/\/api.github.com\/repos\/Q1984\/CVE-2020-16898\/downloads",
    "issues_url":"https:\/\/api.github.com\/repos\/Q1984\/CVE-2020-16898\/issues{\/number}",
    "pulls_url":"https:\/\/api.github.com\/repos\/Q1984\/CVE-2020-16898\/pulls{\/number}",
    "milestones_url":"https:\/\/api.github.com\/repos\/Q1984\/CVE-2020-16898\/milestones{\/number}",
    "notifications_url":"https:\/\/api.github.com\/repos\/Q1984\/CVE-2020-16898\/notifications{?since,all,participating}",
    "labels_url":"https:\/\/api.github.com\/repos\/Q1984\/CVE-2020-16898\/labels{\/name}",
    "releases_url":"https:\/\/api.github.com\/repos\/Q1984\/CVE-2020-16898\/releases{\/id}",
    "deployments_url":"https:\/\/api.github.com\/repos\/Q1984\/CVE-2020-16898\/deployments",
    "created_at":"2020-10-16T06:27:24Z",
    "updated_at":"2020-10-16T11:17:23Z",
    "pushed_at":"2020-10-16T11:17:22Z",
    "git_url":"git:\/\/github.com\/Q1984\/CVE-2020-16898.git",
    "ssh_url":"git@github.com:Q1984\/CVE-2020-16898.git",
    "clone_url":"https:\/\/github.com\/Q1984\/CVE-2020-16898.git",
    "svn_url":"https:\/\/github.com\/Q1984\/CVE-2020-16898",
    "homepage":null,
    "size":36,
    "stargazers_count":0,
    "watchers_count":0,
    "language":null,
    "has_issues":true,
    "has_projects":true,
    "has_downloads":true,
    "has_wiki":true,
    "has_pages":false,
    "forks_count":0,
    "mirror_url":null,
    "archived":false,
    "disabled":false,
    "open_issues_count":0,
    "license":null,
    "forks":0,
    "open_issues":0,
    "watchers":0,
    "default_branch":"main",
    "permissions":{
      "admin":false,
      "maintain":false,
      "push":false,
      "triage":false,
      "pull":true
    },
    "temp_clone_token":"",
    "network_count":0,
    "subscribers_count":1,
    "matched_on":"cve poc pushed:2020-10-15..2020-10-21",
    "matched_at":"2021-08-20T05:36:11Z",
    "organization":null,
    "owner_login":"Q1984",
    "owner_id":50952023,
    "owner_node_id":"MDQ6VXNlcjUwOTUyMDIz",
    "owner_avatar_url":"https:\/\/avatars.githubusercontent.com\/u\/50952023?v=4",
    "owner_gravatar_id":"",
    "owner_url":"https:\/\/api.github.com\/users\/Q1984",
    "owner_html_url":"https:\/\/github.com\/Q1984",
    "owner_followers_url":"https:\/\/api.github.com\/users\/Q1984\/followers",
    "owner_following_url":"https:\/\/api.github.com\/users\/Q1984\/following{\/other_user}",
    "owner_gists_url":"https:\/\/api.github.com\/users\/Q1984\/gists{\/gist_id}",
    "owner_starred_url":"https:\/\/api.github.com\/users\/Q1984\/starred{\/owner}{\/repo}",
    "owner_subscriptions_url":"https:\/\/api.github.com\/users\/Q1984\/subscriptions",
    "owner_organizations_url":"https:\/\/api.github.com\/users\/Q1984\/orgs",
    "owner_repos_url":"https:\/\/api.github.com\/users\/Q1984\/repos",
    "owner_events_url":"https:\/\/api.github.com\/users\/Q1984\/events{\/privacy}",
    "owner_received_events_url":"https:\/\/api.github.com\/users\/Q1984\/received_events",
    "owner_type":"User",
    "owner_site_admin":false
  },
  {
    "id":304755825,
    "node_id":"MDEwOlJlcG9zaXRvcnkzMDQ3NTU4MjU=",
    "name":"cve-2020-16898",
    "full_name":"0xeb-bp\/cve-2020-16898",
    "private":false,
    "html_url":"https:\/\/github.com\/0xeb-bp\/cve-2020-16898",
    "description":"PoC BSOD for CVE-2020-16898 (badneighbor)",
    "fork":false,
    "url":"https:\/\/api.github.com\/repos\/0xeb-bp\/cve-2020-16898",
    "forks_url":"https:\/\/api.github.com\/repos\/0xeb-bp\/cve-2020-16898\/forks",
    "keys_url":"https:\/\/api.github.com\/repos\/0xeb-bp\/cve-2020-16898\/keys{\/key_id}",
    "collaborators_url":"https:\/\/api.github.com\/repos\/0xeb-bp\/cve-2020-16898\/collaborators{\/collaborator}",
    "teams_url":"https:\/\/api.github.com\/repos\/0xeb-bp\/cve-2020-16898\/teams",
    "hooks_url":"https:\/\/api.github.com\/repos\/0xeb-bp\/cve-2020-16898\/hooks",
    "issue_events_url":"https:\/\/api.github.com\/repos\/0xeb-bp\/cve-2020-16898\/issues\/events{\/number}",
    "events_url":"https:\/\/api.github.com\/repos\/0xeb-bp\/cve-2020-16898\/events",
    "assignees_url":"https:\/\/api.github.com\/repos\/0xeb-bp\/cve-2020-16898\/assignees{\/user}",
    "branches_url":"https:\/\/api.github.com\/repos\/0xeb-bp\/cve-2020-16898\/branches{\/branch}",
    "tags_url":"https:\/\/api.github.com\/repos\/0xeb-bp\/cve-2020-16898\/tags",
    "blobs_url":"https:\/\/api.github.com\/repos\/0xeb-bp\/cve-2020-16898\/git\/blobs{\/sha}",
    "git_tags_url":"https:\/\/api.github.com\/repos\/0xeb-bp\/cve-2020-16898\/git\/tags{\/sha}",
    "git_refs_url":"https:\/\/api.github.com\/repos\/0xeb-bp\/cve-2020-16898\/git\/refs{\/sha}",
    "trees_url":"https:\/\/api.github.com\/repos\/0xeb-bp\/cve-2020-16898\/git\/trees{\/sha}",
    "statuses_url":"https:\/\/api.github.com\/repos\/0xeb-bp\/cve-2020-16898\/statuses\/{sha}",
    "languages_url":"https:\/\/api.github.com\/repos\/0xeb-bp\/cve-2020-16898\/languages",
    "stargazers_url":"https:\/\/api.github.com\/repos\/0xeb-bp\/cve-2020-16898\/stargazers",
    "contributors_url":"https:\/\/api.github.com\/repos\/0xeb-bp\/cve-2020-16898\/contributors",
    "subscribers_url":"https:\/\/api.github.com\/repos\/0xeb-bp\/cve-2020-16898\/subscribers",
    "subscription_url":"https:\/\/api.github.com\/repos\/0xeb-bp\/cve-2020-16898\/subscription",
    "commits_url":"https:\/\/api.github.com\/repos\/0xeb-bp\/cve-2020-16898\/commits{\/sha}",
    "git_commits_url":"https:\/\/api.github.com\/repos\/0xeb-bp\/cve-2020-16898\/git\/commits{\/sha}",
    "comments_url":"https:\/\/api.github.com\/repos\/0xeb-bp\/cve-2020-16898\/comments{\/number}",
    "issue_comment_url":"https:\/\/api.github.com\/repos\/0xeb-bp\/cve-2020-16898\/issues\/comments{\/number}",
    "contents_url":"https:\/\/api.github.com\/repos\/0xeb-bp\/cve-2020-16898\/contents\/{+path}",
    "compare_url":"https:\/\/api.github.com\/repos\/0xeb-bp\/cve-2020-16898\/compare\/{base}...{head}",
    "merges_url":"https:\/\/api.github.com\/repos\/0xeb-bp\/cve-2020-16898\/merges",
    "archive_url":"https:\/\/api.github.com\/repos\/0xeb-bp\/cve-2020-16898\/{archive_format}{\/ref}",
    "downloads_url":"https:\/\/api.github.com\/repos\/0xeb-bp\/cve-2020-16898\/downloads",
    "issues_url":"https:\/\/api.github.com\/repos\/0xeb-bp\/cve-2020-16898\/issues{\/number}",
    "pulls_url":"https:\/\/api.github.com\/repos\/0xeb-bp\/cve-2020-16898\/pulls{\/number}",
    "milestones_url":"https:\/\/api.github.com\/repos\/0xeb-bp\/cve-2020-16898\/milestones{\/number}",
    "notifications_url":"https:\/\/api.github.com\/repos\/0xeb-bp\/cve-2020-16898\/notifications{?since,all,participating}",
    "labels_url":"https:\/\/api.github.com\/repos\/0xeb-bp\/cve-2020-16898\/labels{\/name}",
    "releases_url":"https:\/\/api.github.com\/repos\/0xeb-bp\/cve-2020-16898\/releases{\/id}",
    "deployments_url":"https:\/\/api.github.com\/repos\/0xeb-bp\/cve-2020-16898\/deployments",
    "created_at":"2020-10-16T23:03:15Z",
    "updated_at":"2021-08-19T06:09:27Z",
    "pushed_at":"2020-10-16T23:12:52Z",
    "git_url":"git:\/\/github.com\/0xeb-bp\/cve-2020-16898.git",
    "ssh_url":"git@github.com:0xeb-bp\/cve-2020-16898.git",
    "clone_url":"https:\/\/github.com\/0xeb-bp\/cve-2020-16898.git",
    "svn_url":"https:\/\/github.com\/0xeb-bp\/cve-2020-16898",
    "homepage":null,
    "size":2,
    "stargazers_count":21,
    "watchers_count":21,
    "language":"Python",
    "has_issues":true,
    "has_projects":true,
    "has_downloads":true,
    "has_wiki":true,
    "has_pages":false,
    "forks_count":5,
    "mirror_url":null,
    "archived":false,
    "disabled":false,
    "open_issues_count":0,
    "license":null,
    "forks":5,
    "open_issues":0,
    "watchers":21,
    "default_branch":"main",
    "permissions":{
      "admin":false,
      "maintain":false,
      "push":false,
      "triage":false,
      "pull":true
    },
    "temp_clone_token":"",
    "network_count":5,
    "subscribers_count":2,
    "matched_on":"cve poc pushed:2020-10-15..2020-10-21",
    "matched_at":"2021-08-20T05:36:11Z",
    "organization":null,
    "owner_login":"0xeb-bp",
    "owner_id":53204152,
    "owner_node_id":"MDQ6VXNlcjUzMjA0MTUy",
    "owner_avatar_url":"https:\/\/avatars.githubusercontent.com\/u\/53204152?v=4",
    "owner_gravatar_id":"",
    "owner_url":"https:\/\/api.github.com\/users\/0xeb-bp",
    "owner_html_url":"https:\/\/github.com\/0xeb-bp",
    "owner_followers_url":"https:\/\/api.github.com\/users\/0xeb-bp\/followers",
    "owner_following_url":"https:\/\/api.github.com\/users\/0xeb-bp\/following{\/other_user}",
    "owner_gists_url":"https:\/\/api.github.com\/users\/0xeb-bp\/gists{\/gist_id}",
    "owner_starred_url":"https:\/\/api.github.com\/users\/0xeb-bp\/starred{\/owner}{\/repo}",
    "owner_subscriptions_url":"https:\/\/api.github.com\/users\/0xeb-bp\/subscriptions",
    "owner_organizations_url":"https:\/\/api.github.com\/users\/0xeb-bp\/orgs",
    "owner_repos_url":"https:\/\/api.github.com\/users\/0xeb-bp\/repos",
    "owner_events_url":"https:\/\/api.github.com\/users\/0xeb-bp\/events{\/privacy}",
    "owner_received_events_url":"https:\/\/api.github.com\/users\/0xeb-bp\/received_events",
    "owner_type":"User",
    "owner_site_admin":false
  }
]
